src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlElasticPoolActivity.md

```diff
@@ -0,0 +1,170 @@
+﻿---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
```

Comment:
@payiAzure the build will fail because there is no online help URL found here. It should be `https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/stop-azurermsqlelasticpoolactivity`

Reply:
@cormacpayne adding the help URL